medusa/cassandra_utils.py

```diff
@@ -619,11 +632,14 @@ def is_open(host, port):
         s.shutdown(socket.SHUT_RDWR)
         is_accessible = True
     except socket.error as e:
-        logging.debug('Port {} close on host {}'.format(port, host), exc_info=e)
-        is_accessible = False
+        logging.error('Port {} close on host {}'.format(port, host), exc_info=e)
     finally:
-        s.close()
-        return is_accessible
+        try:
+            s.close()
+        except os.error as ose:
+            logging.error('Port {} close on host {}'.format(port, host), exc_info=ose)
```

Comment:
```suggestion
        s.close()
```

After checking, this call cannot fail so we don't need to wrap it in a try/except block.